src/texture.h

```diff
@@ -69,8 +69,8 @@ class ImageTexture: public Texture
 
 Color ImageTexture::value(float u, float v, const Vec3& p) const 
 {
-    int i = u * nx;
-    int j = (1 - v) * ny - 0.001f;
+    int i = static_cast<int>(u * nx);
+    int j = static_cast<int>((1 - v) * ny - 0.001f);
```

Comment:
I was also thinking of using `std::floor` but I'm not really sure if that is faster or if we even want that kind of precision.

Reply:
Anyway, I appreciate your effort in removing warnings! :D